### DISCUSSION TITLE
Custom key value pair to useHead.script

### DISCUSSION THREAD

**Discussion by procrates** [nuxt/nuxt#19641] [Questions] [answered]
I need to add this as a script:

```
<script
  async
  data-environment="playground"
  src="https://osm.klarnaservices.com/lib.js"
  data-client-id="id"
></script>
```
Looking through the code but cant find out a way to add custom key, value pairs. Maybe something i am missing.

Trying this, but dosnt work.
```
useHead({
  script: [
    {
      async: true,
      dataEnvironment: "playground", 
      dataClientId: "id",
      src: "https://osm.klarnaservices.com/lib.js"
    }
  ]
})
```

Thanks for any help:)

**Comment by Baroshem** [comment]
Hey,

you should be able to add the script like you added here but not sure if you can add more unknown properties like this 

The head support actually comes from unhead package that you can check below.
https://unhead.harlanzw.com/

Maybe this could help you as well.

Or, you could use the Script component as shown here
https://nuxt.com/docs/getting-started/seo-meta#components

**Reply by procrates** [reply]
That is my point, how to add those custom values. As there are no documentations about it in the module docs I was checking the code, but cant find the properties that can be added.

Also there are are no <Script> component anymore in the docs.

Thanks for helping out thou:)

**Comment by procrates** [comment, marked as answer]
This actually works. Forgetting basic js...
```
useHead({
  script: [
    { src: 'https://osm.klarnaservices.com/lib.js', "data-environment": 'env', "data-client-id": "id", defer: true, async: true }
  ]
})
```
